README.md

```diff
@@ -1,19 +1,22 @@
-# reddit_viewer
+# Tabris.js Reddit Viewer Example
 
-## Run
+[![GitPod Logo](images/run-in-gitpod.png)](https://gitpod.io/#https://github.com/eclipsesource/tabris-js-reddit-viewer)
 
-In the project directory, run the [Tabris CLI](https://www.npmjs.com/package/tabris-cli) command:
+## Description
+This is an example Tabris.js project that demonstrates how to use the ListView feature. The app shows Reddit posts, enables them to be filtered and to change the display mode. The subreddits are retrieved from the Reddit API and rendered by ListView.
```

Comment:
Hm. The ListView is mentioned twice. I think mentioning it below is enough. Here you should specific the project type, i.e. "a TypeScript/JSX-based Tabris.js example project."